sunpy/net/dataretriever/client.py

```diff
@@ -285,15 +238,16 @@ def search(self, *args, **kwargs):
         """
         baseurl, pattern, matchdict = self.pre_search_hook(*args, **kwargs)
         scraper = Scraper(baseurl, regex=True)
-        filesmeta = scraper._extract_files_meta(matchdict['Time'], extractor=pattern,
+        tr = TimeRange(matchdict['Start Time'], matchdict['End Time'])
+        filesmeta = scraper._extract_files_meta(tr, extractor=pattern,
                                                 matcher=matchdict)
         metalist = []
         for i in filesmeta:
             rowdict = self.post_search_hook(i, matchdict)
             metalist.append(rowdict)
         return QueryResponse(metalist, client=self)
 
-    def fetch(self, qres, path=None, overwrite=False,
+    def fetch(self, qres, *, path=None, overwrite=False,
```

Comment:
Converting these arguments to keyword only needs a breaking changelog entry

Reply:
```suggestion
    def fetch(self, qres, path=None, overwrite=False,
```